ui/components/src/utils/theme-gen.ts

```diff
@@ -22,12 +22,7 @@ type MuiTheme = Omit<Theme, 'components'>;
 
 export function generateChartsTheme(
   muiTheme: MuiTheme,
-  echartsTheme: EChartsTheme,
-  /**
-   * The id of the container that will have the chart tooltip appended to it.
-   * By default, chart tooltip uses the body of the top-level document object.
-   */
-  tooltipPortalContainerId?: string
```

Comment:
Do we need to remove this, this was part of a bugfix: https://github.com/perses/perses/pull/1309, does this still the case?